policyengine_us/variables/gov/ssa/ssi/eligibility/status/is_ssi_ineligible_spouse.py

```diff
@@ -10,7 +10,21 @@ class is_ssi_ineligible_spouse(Variable):
     reference = "https://www.law.cornell.edu/uscode/text/42/1382c#b"
 
     def formula(person, period, parameters):
-        spouse = person("is_tax_unit_spouse", period)
+        # Consider both tax unit spouse and tax unit head as potential spouses
+        is_spouse = person("is_tax_unit_head_or_spouse", period)
+
+        # Check if they're in a marital unit
```

Comment:
what's the purpose of this; won't it always be true? if they're themselves a head or spouse, their marital unit will have a head or spouse